step-templates/aws-linux-install-tentacle.json

```diff
@@ -102,7 +102,7 @@
       "Name": "instrallTentacle.octoServerUrl",
       "Label": "Octo Server Url",
       "HelpText": "The server url to register the tentacle with.  Defaults to the base url",
-      "DefaultValue": "#{Octopus.Web.BaseUrl}",
+      "DefaultValue": "#{#{if Octopus.Web.ServerUri}#{Octopus.Web.ServerUri}#{else}#{Octopus.Web.BaseUrl}#{/if}}",
       "DisplaySettings": {
```

Comment:
I think the outer `#{` and `}` in these parameters (all steps like this) is superfluous, and might not evaluate as you might expect?

Reply:
You're right. Find/Replace went bad on 10 of the files.